index.html

```diff
@@ -288,13 +421,44 @@ <h4>
                     <li><img class="arrow2" src=img/right_arrow.png alt="right_arrow"></li>
                 </ul>
 
-                <img src="img/Separator.png" id="last_seperator" alt="seperator">
+                <img src="img/Separator.png" class="last_seperator" alt="seperator">
 
         </div>
-        </section>
+        <div class="works_desktop">
+            <ul class="language">
+              <li><p class="slider_titles">Languages</p></li>
+              <li class="down_arrow"><img class="arrow" src="img/down_arrow.png" alt="down_arrow"></li>
+            </ul>
+
+            <div class="abilities">
+                <ul class="languages">
+                    <li><img class="language_img" src="img/js_icon_desktop.svg" alt="javascript"></li>
+                    <li><img class="language_img" src="img/html_icon_desktop.svg" alt="html"></li>
+                    <li><img class="language_img" src="img/css_icon_desktop.svg" alt="css"></li>
+                </ul>
+            </div>
+
+            <!-- <img src="img/Separator.png" alt="seperator"> -->
+
+            <ul class="Frameworks">
+                <li><p class="slider_titles">Frameworks</p></li>
+                <li><img class="arrow" src=img/right_arrow.png alt="right_arrow"></li>
+            </ul>
+
+            <img src="img/desktop_seperator.svg" alt="seperator">
+
+            <ul class="Skills">
+                <li><p class="slider_titles">Skills</p></li>
+                <li><img class="arrow2" src=img/right_arrow.png alt="right_arrow"></li>
+            </ul>
+
+            <img src="img/desktop_seperator.svg" class="last_seperator" alt="seperator">
+        </div>
+    </section>
+
         <section class="contact_form">
             <h3 class="contact_header">Contact Me</h3>
```

Comment:
- [ ] Please fix also you `contact-form` section for desktop :upside_down_face: :upside_down_face: 

Your design

![Screenshot from 2022-07-06 13-51-52](https://user-images.githubusercontent.com/74501523/177547247-716dcc5b-a52e-463a-9c47-9a980c5cc125.png)

Figma design

![Screenshot from 2022-07-06 13-52-13](https://user-images.githubusercontent.com/74501523/177547312-42c39785-44e8-4691-9f9a-985bcd7de938.png)